editor/editor/mode/text.js

```diff
@@ -1,11 +1,20 @@
+/**
+ * External dependencies
+ */
+import Textarea from 'react-textarea-autosize';
```

Comment:
I retract the above, our Webpack configuration should handle this automatically since we define React as an external:

https://github.com/WordPress/gutenberg/blob/af5e33d/webpack.config.js#L21-L24